Lib/glyphsLib/builder/glyph.py

```diff
@@ -146,6 +146,20 @@ def to_ufo_glyph(self, ufo_glyph, layer, glyph, do_color_layers=True):  # noqa:
         self.to_ufo_layer_user_data(ufo_glyph, layer)  # .user_data
         self.to_ufo_smart_component_axes(ufo_glyph, glyph)  # .components
 
+    # Optimization: profiling glyphs2ufo of NotoSans-MM.glyphs (6000 glyphs) on a Mac
+    # mini late 2014, Python 3.6.8, revealed that a whopping 17% of the time was spent
+    # converting lastChange to UFO timestamps. I could not reproduce this on a Windows
+    # 10/Python 3.7 setup, so this might be a platform thing. If-guarding anyway
+    # because these timestamps are useless in a UFO scenario if you use Git.
+
+    # This *should* be gated with not self.minimal, but that breaks regression tests
```

Comment:
regression tests breaking is fine when the change is desired (will auto-fix themselves upon merging to main but need to be reviewed manually and accepted as good).
But maybe it's fine to do this change in a separate PR